### ISSUE TITLE
Suggestion: Let me create an interactive window attached to the same kernel as a notebook

### ISSUE THREAD

**Issue by rchiodo** [microsoft/vscode-jupyter#4543]
This would allow me to experiment with data in the kernel without having to muck up my notebook.

**Comment by rchiodo** [comment]
At a minimum we could surface the connect_info that allows a user to run an ipython console:
https://jupyter-notebook.readthedocs.io/en/4.x/examples/Notebook/rstversions/Connecting%20with%20the%20Qt%20Console.html#manual-connection

**Comment by DonJayamanne** [comment]
Blocked on https://github.com/microsoft/vscode-jupyter/issues/9517

**Comment by amunger** [comment]
closing in favor of https://github.com/microsoft/vscode-jupyter/issues/4573
